src/docs/ftw-content/hosted-translations/index.md

```diff
@@ -0,0 +1,236 @@
+more:_
+
+- _[Translations in Flex Console](/concepts/translations/)_
+- _[How built-in translations work in the FTW templates](ftw/how-to-change-ftw-bundled-ui-translations/)_
+
+## Hosted translations
+
+**Hosted translations** are translations that the marketplace operator
+can edit in Flex Console. The client app then needs to fetch them using
+the Asset Delivery API.
+
+It is good to note that even if the operator adds some hosted
+translations using Flex Console, the FTW template still needs to have a
+built-in translation file for the translation keys that do not have a
+value in the hosted asset. That way, the UI can still render something
+meaningful for the parts of the page that the operator has not modified.
+
+There are two ways to fetch translation assets using Asset Delivery API:
+by version or by alias.
+
+#### Fetching translations by version
+
+All assets are identifiable by their version, and versions are
+immutable. Therefore if you fetch assets by version, they can be cached
+for an extended period of time. Read more about [caching assets]() TODO:
+LINK. When fetching translations by version, they are cached for an
+extended period of time, which helps to avoid unnecessary data loading.
+
+Since FTW templates use Flex SDK, the response always contains the data
+for the requested asset, even if the asset has not changed in the
+specified version. However, when using the API endpoint without SDK, it
+is important to take into account the API's [asset redirect behavior]()
+TODO: LINK.
+
+```js
+sdk.assetByVersion({
+  path: 'content/translations.json',
+  version: '<some-hash-string>',
+});
+```
+
+#### Fetching translations by alias
+
+In addition to fetching assets by version, you can fetch them by a
+specific alias instead of a version. Currently, translations can be
+fetched with the alias `latest`, which returns the most recently updated
+version of the translations. The response also contains the version
+information for the most recent asset, so that subsequent fetches can be
+done based on asset version.
+
+When fetching by alias, the cache time is 5 seconds for development
+environment and up to 5 minutes for production environment. In other
+words, it can take up to 5 minutes for translation updates to be visible
+in a production environment. These cache times are subject to change.
+
+```js
+sdk.assetByAlias({
+  path: 'content/translations.json',
+  alias: 'latest',
+});
+```
+
+We have added a new global Redux file
+(_src/ducks/hostedAssets.duck.js_), which exports Redux Thunk function
+called _fetchAppAssets_. This function actually makes the calls to the
+Asset Delivery API.
+
+Translations are part of app-wide hosted assets that have configuration
+in _src/config.js_
+
+```js
+// CDN assets for the app. Configurable through Flex Console.
+// Currently, only translation.json is available.
+const appCdnAssets = {
+  translations: 'content/translations.json',
+};
+```
```

Comment:
This is used by both SDK calls (alias and versioned), so, it's probably better to tell this already before these sub headings.

----

**Hosted translations** are translations that the marketplace operator
can edit in Flex Console. The client app then needs to fetch them using
the Asset Delivery API.

It is good to note that even if the operator adds some hosted
translations using Flex Console, the FTW template still needs to have a
built-in translation file for the translation keys that do not have a
value in the hosted asset. That way, the UI can still render something
meaningful for the parts of the page that the operator has not modified.

FTW templates have specified hosted translations as part of the app-wide configuration
in _src/config.js_.

```js
// CDN assets for the app. Configurable through Flex Console.
// Currently, only translation.json is available.
// Note: the path must match the path defined in Asset Delivery API
const appCdnAssets = {
  translations: 'content/translations.json',
};
```

In addition, they have added a new global Redux file
(_src/ducks/hostedAssets.duck.js_), which exports the Redux Thunk function
called _fetchAppAssets_. This function actually makes the calls to the
Asset Delivery API.

There are two ways to fetch translation assets using Asset Delivery API:
by the version or by alias.